lib/pages/chat/chat.dart

```diff
@@ -117,6 +117,8 @@ class ChatController extends State<Chat>
 
   PinnedEventsController pinnedEventsController = PinnedEventsController();
 
+  static const waitForJumpToFirstUnreadEvent = Duration(seconds: 1);
```

Comment:
what happen if it take more time than that?

Reply:
I think no need it.